independent-publisher-connectors/Clockify/apiDefinition.swagger.json

```diff
@@ -20,40 +20,260 @@
+          },
+          {
+            "description": "email",
+            "in": "query",
+            "name": "email",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Email",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "description": "projectid",
+            "in": "query",
+            "name": "projectid",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Project Id",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "description": "name",
+            "in": "query",
+            "name": "name",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Name",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "description": "status",
+            "in": "query",
+            "name": "status",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Status",
+            "enum": [
+              "PENDING",
+              "ACTIVE",
+              "DECLINED",
+              "INACTIVE"
+            ],
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "UserList",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "string",
+                    "x-ms-summary": "User Id"
                   },
                   "name": {
                     "type": "string",
                     "x-ms-summary": "Name"
+                  },
+                  "email": {
+                    "type": "string",
+                    "x-ms-summary": "Email"
                   }
                 }
               }
             }
+          },
+          "403": {
+            "description": "Invalid Request",
+            "schema": {
+              "type": "string"
+            }
           }
         }
       }
     },
     "/workspaces/{workspace_id}/clients": {
       "get": {
-        "summary": "List Clients",
+        "summary": "List Clients (deprecated)",
         "description": "List Clients",
         "operationId": "GetClients",
+        "deprecated": true,
+        "x-ms-api-annotation": {
+          "family": "GetClients",
+          "revision": 0
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/WorkspaceParamForPath"
+          },
+          {
+            "description": "Set to true to only return archived clients Returns active clients when false.",
+            "in": "query",
+            "name": "archived",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Archived",
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
```

Comment:
Hello @nk-gears 

The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/GetClients/responses/200/schema/type

can you please confirm that no existing flows or users will be affected by this change?

Reply:
The type was supposed to be array. It's a bug fix. For some reason the initial version had a bug and we just noticed today. So, there should be any issues

Reply:
> The type was supposed to be array. It's a bug fix. For some reason the initial version had a bug and we just noticed today. So, there should be any issues

Can you please confirm that it will not break any existing flows?

Reply:
I have added few fixes now. It should not have any issues for the existing flow.

Reply:
> > The type was supposed to be array. It's a bug fix. For some reason the initial version had a bug and we just noticed today. So, there should be any issues
> 
> Can you please confirm that it will not break any existing flows?

I have added some fixes. It should not break the existing